piptools/scripts/options.py

```diff
@@ -364,3 +370,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_for_distributions",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_DISTRIBUTIONS),
+    help="Name of a distribution to install build dependencies for. "
```

Comment:
```suggestion
    help="Name of a distribution package build environment to install dependencies for. "
```

Reply:
Why do you think the proposed change is an improvement?

I worry that the sequence _distribution package_ may make readers think of [this](https://packaging.python.org/en/latest/glossary/#term-Distribution-Package) but I’m guessing that is not what you refer to.

I could get onboard with renaming _distribution_ to _build environment_ and then this would read _Name of a build environment to install dependencies for_.

Reply:
> I could get onboard with renaming _distribution_ to _build environment_ and then this would read _Name of a build environment to install dependencies for_.

Nice! That does sound more clear. Go for it! Naming things is hard :)

Also: maybe "build target" instead of "build environment"?